melodic/distribution.yaml

```diff
@@ -2734,6 +2734,17 @@ repositories:
       url: https://github.com/stonier/py_trees.git
       version: devel
     status: maintained
+  py_trees_msgs:
+    doc:
+      type: git
+      url: https://github.com/stonier/py_trees_msgs.git
+      version: release/0.3-melodic
+    release:
+      tags:
+        release: release/melodic/{package}/{version}
+      url: https://github.com/stonier/py_trees_msgs-release.git
+      version: 0.3.5-0
+    status: maintained
```

Comment:
Is it possible to add a source entry as well ? (similar to what is done for other distros https://github.com/ros/rosdistro/blob/d9c5069f246761fb8a00fb2664c29a10ac7cb683/kinetic/distribution.yaml#L8352-L8355)